skeleton/frontend/src/Frontend.hs

```diff
@@ -30,9 +30,9 @@ frontend = Frontend
       el "h1" $ text "Welcome to Obelisk!"
       el "p" $ text $ T.pack commonStuff
       
-      -- 'prerender' and 'prerender_' let you choose a widget to run on the server
+      -- `prerender` and `prerender_` let you choose a widget to run on the server
       -- during prerendering and a different widget to run on the client with
-      -- JavaScript. The following will generate a 'blank' widget on the server and
+      -- JavaScript. The following will generate a `blank` widget on the server and
```

Comment:
Using `'` is the "Haddock" way to refer to a function in scope. This doesn't get built by haddock though... still I kinda prefer the haddock-friendliness. Just FYI really.